cmd/cmd_test.go

```diff
@@ -2,19 +2,25 @@ package cmd
 
 import (
 	"testing"
-
-	"github.com/bmizerany/assert"
 )
 
 func TestNew(t *testing.T) {
 	execCmd := New("vim --noplugin")
-	assert.Equal(t, "vim --noplugin", execCmd.Name)
```

Comment:
@lufia Thanks for suggesting this!

To avoid having to update many tests (which in general is always risky), could we find another library that offers the same interface, or roll out our own mini-package that simply implements the assertion methods we need? That way we could simply switch the import statements and not have to update any code.

Also, I don't feel the pressing need to migrate away from bmizerany/assert right now, unless the repository is getting deleted. It works well and we don't need it to gain any new features. 

Let me know what you think!

Reply:
@mislav Thank you for your reply.

My motivation was I want to upgrade all dependencies.

However the command `go get -u all` don't upgrade github.com/kr/pty@v1.1.1 to v1.1.8.

- github.com/bmizerany/assert  is depended on kr/pretty
- github.com/kr/pretty is depended on kr/text@v0.1.0
- github.com/kr/text@v0.1.0 is depended on kr/pty@v1.1.1

I read that bmizerany/assert is no longer maintained. Moreover github.com/kr/pty is archived. Thus I thinked replacing bmizerany/assert is better than keeping it.

Reply:
Thanks for that info!

> or roll out our own mini-package that simply implements the assertion methods we need? That way we could simply switch the import statements and not have to update any code.

What do you think about this proposal? I think that it would be my preferred solution and wouldn't require changing any tests. Let me know if you would be able to do it in this PR.

You could put the package under `internal/assert/`.

Reply:
@mislav I'll try it!